### PR TITLE
Assign company tags components

### PR DESCRIPTION
**Pull request by Hyperkid123**
Tagging component for users iin RBAC.

stprybook: https://5b72b6cc1f12b728ed3c03db--confident-banach-ee0a7a.netlify.com/?selectedKind=Rbac%20forms&selectedStory=Rbac%20assign%20user%20tags&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Fstories%2Fstories-panel

### CODE REVIEW

**Comment by Hyperkid123** [comment]
@miq-bot assign @karelhala

**Comment by Hyperkid123** [comment]
@miq-bot remove-label unmergeable

File: src/rbac-forms/rbacAssignUserTags.jsx [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+      categoryEntries: {},
+      selectedEntries: {},
+      isLoaded: false,
+      initialAssignments: {},
+    };
+  }
+
+  componentDidMount() {
+    const { users, categories } = this.props;
+    const commonTags = intersectionWith((a, b) => (a.id === b.id), ...users.map(user => user.tags));
```
Comment (karelhala) [inline src/rbac-forms/rbacAssignUserTags.jsx]: Can you please use better names for variables then `a` and `b`?

File: src/rbac-forms/rbacAssignUserTags.jsx [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+    const tagsNames = {};
+    const selectedEntries = {};
+    commonTags.forEach((tag) => {
+      tagsNames[tag.name.split('/')[2]] = { value: tag.id };
+    });
+    Object.keys(tagsNames).forEach((categoryName) => {
+      selectedEntries[categories.find(category => category.name === categoryName).value] = { ...tagsNames[categoryName] };
```
Comment (karelhala) [inline src/rbac-forms/rbacAssignUserTags.jsx]: Can you please merge these two foreaches together? I mean isn't `tag.name.split('/')[2]` same as `categoryName`?

**Pull request merged**